src/blackjack.js

```diff
@@ -34,7 +53,15 @@ function shouldSplit(cardOne, cardTwo) {
  * @return {Boolean}
  */
 function shouldHit(cardOne, cardTwo) {
-
+  if (sum(cardValue(cardOne), cardValue(cardTwo)) < 17) {
```

Comment:
Put this AFTER checking for splits. I think right now this would hit on a 5, 5 combo since that would be less than 17 and return right away.